.github/workflows/windows.yml

```
@@ -13,7 +13,7 @@ concurrency:
 
 jobs:
   test:
-    name: "${{ matrix.os }}/${{ matrix.arch }} (${{ matrix.generator }})"
+    name: "${{ matrix.os }}/${{ matrix.arch }}/shared ${{ matrix.shared }} (${{ matrix.generator }})"
```

I think formatting this as  `windows-2022/ARM64 (Visual Studio 17 2022, shared)`/`windows-2019/Win32 (Visual Studio 16 2019` could make it easier to understand. Maybe something like this would work?
```suggestion
    name: "${{ matrix.os }}/${{ matrix.arch }} (${{ matrix.generator }}${{ matrix.shared == 'ON' && ', shared' || '' }})"
```

---

Sure, let's give it a try.